packages/test/src/test-integration-update.ts

```diff
@@ -3,14 +3,39 @@ import { isGrpcServiceError } from '@temporalio/client';
 import * as wf from '@temporalio/workflow';
 import { helpers, makeTestFunction } from './helpers-integration';
 
-const test = makeTestFunction({ workflowsPath: __filename });
+// Use a reduced server long-poll expiration timeout, in order to confirm that client
+// polling/retry strategies result in the expected behavior, in particular:
+// - An update that is cut off by this timeout but eventually completes should be seen by the client
+//   to complete eventually and seamlessly.
+// - For an update that never completes (e.g. because worker is offline), startUpdate never returns
+//   a handle to the client.
+const LONG_POLL_EXPIRATION_INTERVAL_SECONDS = 1.0;
```

Comment:
We could reduce this further, but currently this will fail due to a server bug: the server should allow `UpdateWorkflowExecution` requests against completed workflows: https://github.com/temporalio/temporal/pull/6083